RELEASE_NOTES.md

```diff
@@ -1,9 +1,10 @@
-# Version <Next Release>
+# Version 0.5.10
```

Comment:
Are we squashing this release with 0.5.9? - Be sure to keep setup.py in sync if you change here.

Reply:
We are not squashing anymore.  Tim would like one version per ticket and has requested that this ticket be 0.5.10.  setup.py is in sync.